service/src/main/resources/config/verily/sdd/expanded/entity/genotyping.json

```diff
@@ -306,7 +330,34 @@
     "textSearchMapping" : {
       "attributes" : [ "id" ]
     },
-    "hierarchyMappings" : { }
+    "hierarchyMappings" : {
+      "standard" : {
+        "childParent" : {
+          "tablePointer" : {
+            "rawSql" : "SELECT /* Use parent ids defined in platform.sql */     parent_seq + 100 AS parent,     CAST(platform_id as INT64) AS child FROM `victr-tanagra-test.sd_static.genotype_criteria` g, `victr-tanagra-test.sd_static.platform` p WHERE g.name = p.assay_name "
+          },
+          "fieldPointers" : {
+            "parent" : {
+              "column" : "parent"
+            },
+            "child" : {
+              "column" : "child"
+            }
+          }
+        },
+        "rootNodesFilter" : {
+          "tablePointer" : {
+            "rawSql" : "/* Match the platform ids in platform.sql */ SELECT 101 AS id UNION ALL SELECT 102 UNION ALL SELECT 103 "
+          },
+          "fieldPointers" : {
+            "id" : {
+              "column" : "id"
+            }
+          }
+        },
+        "maxHierarchyDepth" : 1
```

Comment:
I would have expected 2 here since there are two levels, but maybe this is more technically the number of passes of the hierarchy algorithm?

Reply:
Interesting. I think `maxHierarchyDepth=1` makes sense (or at least is one possible interpretation). If it was `maxHierarchyLevels`, that would be 2.